changelog.md

```diff
@@ -2,6 +2,9 @@
 
 Please append a line to the changelog for each change made.
 
+## v1.2
+* Mapping rules json structural change, now objects associated to a CDM table are a dictionary (with a key name) rather than just a list.
+
```

Comment:
@spco is this fine?

Reply:
It needs appending to v1.1-beta list please

Reply:
that makes a lot more sense - slow morning -  thanks, done